test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_1_IT.java

```diff
@@ -425,15 +425,19 @@ private void verifyCompactionCommitAndDead(FateStore<Manager> fateStore, TableId
       FateId fateId, List<ExternalCompactionId> cids) {
     var ctx = getCluster().getServerContext();
 
+    System.out.println("cids:" + cids);
```

Comment:
Alternatively, can use slf4j logger in test code like we do in non-test code. It looks like maven-surefire-plugin uses slf4j-simple to handle slf4j logging from tests and collect them in the test output files, but print statements like these will probably just land on the regular console in the middle of the build. We do configure surefire to redirect STDOUT to a file, but it's probably better to just use logging.

This isn't really important here, though... we have tests that do both, and we aren't very consistent about it. So, either is fine today.

Reply:
Also get a timestamp w/ logging and its more consistent w/ the other logged output. Switched the new print and two existing prints to use logging.